pubspec.yaml

```diff
@@ -20,3 +20,8 @@ dev_dependencies:
   matcher: ^0.12.0
   path: '>=0.9.0 <2.0.0'
   test: ^0.12.24+1
+dependency_overrides:
```

Comment:
I don't think we want to use overrides. We'll need to publish a new version of analyzer and update the dependency.

Reply:
👍

Reply:
Alternatively you could consider making a reference to an analyzer git commit

https://www.dartlang.org/tools/pub/dependencies#git-packages

I would still not commit this but it would at least give us an idea of whether the build would be green.